crates/cairo-profiler/src/sierra_loader.rs

```diff
@@ -11,134 +11,143 @@ use trace_data::{CallTrace, CallTraceNode};
 
 /// Map with sierra and casm debug info needed for function level profiling.
 /// All paths in the map are absolute paths.
-pub struct CompiledArtifactsPathMap(HashMap<Utf8PathBuf, CompiledArtifacts>);
+pub struct CompiledArtifactsCache(HashMap<Utf8PathBuf, CompiledArtifacts>);
```

Comment:
I am not sure if cache is good name, it suggests some eviction strategy etc. this is just a map

Reply:
It is an in memory cache, can change to `CompiledArtifactsInMemoryCache`. This struct exists for performance reasons, imo this name tells the purpose better

Reply:
I would leave it as it is